crates/hir_expand/src/ast_id_map.rs

```diff
@@ -117,6 +145,12 @@ impl AstIdMap {
     }
 }
 
+fn hash_ptr(ptr: &SyntaxNodePtr) -> u64 {
+    let mut hasher = BuildHasherDefault::<FxHasher>::default().build_hasher();
```

Comment:
Just looking at this from the weekly feature report, but isn't this line equivalent to `let mut hasher = FxHasher::default();`?

Reply:
Probably! I never really wrapped my head around build hashed thing :-)